static/js/event-handlers.js

```diff
@@ -589,6 +589,20 @@ function logOptionSingleHandler() {
         if (colDef.field === "logs"){
             colDef.cellStyle = null;
             colDef.autoHeight = null;
+            colDef.cellRenderer = function(params) {
+                const data = params.data || {};
+                let logString = '';
+                let counter = 0;
```

Comment:
can this be boolean flag instead of counter?

Reply:
Done.